### ISSUE TITLE
Incorporate Poll Delay in Appium Fluent Wait

### ISSUE THREAD

**Issue by AlessandroMiccoli** [appium/java-client#2111]
## Description

I need a poll delay before starting to look for an element. At the moment, I didn't find this feature in`AppiumFluentWait.`

## Environment

* Java client build version: 8.5.1

## Current Solution

When I need a poll delay, I use the library `awaitility`, which provides this feature and others. Or I find an alternative solution using `TimeUnit#sleep`.

## Code

I would like to have something like this:

AppiumFluentWait<>(this.driver)**.pollDelay(pollDelay)**. etc...

## Questions:

1. Is there a reason for this missing feature?
2. What is the right way to do it, even with the last version of the appium java-client library?
3. If there isn't a solution with the library, is it something you are interested in having it? Probably, I could find some time to integrate it.

**Comment by valfirst** [comment]
@AlessandroMiccoli please feel free to create PR

**Comment by AlessandroMiccoli** [comment]
@valfirst I created the [PR 2116](https://github.com/appium/java-client/pull/2116)

I see a lot of new features we could implement (i.e. setDefaultValues), but probably the best place is directly in Selenium method `FluentWait`. What do you think? 

I just implemented the `pollDelay`, so I was not out of scope.
